pennylane/devices/__init__.py

```diff
@@ -34,6 +34,7 @@
     default_gaussian
     default_mixed
     default_qutrit
+    default_qutrit_mixed
```

Comment:
Thanks, can you also add the `DefaultQutritMixed` class to the "Next generation devices" section below under `NullQubit`?

Reply:
Also I am wondering if the position I put `default_qutrit_mixed` works for you @mudit2812?